nc-news-project/src/components/Home.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react"
+import {getArticles} from '../api'
+import {Link} from 'react-router-dom'
+import moment from 'moment'
+import PropgateLoader from 'react-spinners/PropagateLoader'
+
+const Home = () => {
+    const [articleList, setArticleList] = useState([{}])
+    const [isLoading, setIsLoading] = useState(true)
+
+   useEffect(() => {
+    getArticles().then(data => {
+        data.articles.map(article => {
+            let str = article.created_at
+            let date = moment(str).format("l")
+            return article.created_at = date
+        })
+        setArticleList(data.articles)
+        setIsLoading(false)
+    })
+   }, [articleList])
+
+   const style = {position: "fixed", top: "50%", left: "50%", transform: "translate(-50%, -50%)" }
+
+   if(isLoading) return <div style={style}><PropgateLoader /></div>
+    return (
+        <ul className="article-list" key='article-list'>
+            {articleList.map(article => {
+                return (
+                    <div className="single-article" >
```

Comment:
The only child of a <ul> should be an <li> and the parent of an <li> should be either <ul> or <ol>, here you are setting the child to be a <div> with <li>'s inside of it. I would change the div to be the <li>. Then using semantic html, we should set any text to be a <p> tag